frontend/src/layout/navigation-3000/sidepanel/SidePanel.scss

```diff
@@ -20,7 +20,7 @@
             box-shadow: 0px 0px 30px rgba(0, 0, 0, 0.2);
 
             [theme='dark'] & {
-                box-shadow: 0px 0px 30px rgba(255, 255, 255, 0.1);
+                box-shadow: none;
```

Comment:
I feel like this changes the problem rather than solving it but sure.

Reply:
I did a bit of a dive on how others got around this. Most suggested using lighter coloured background to signify elevation in dark mode. That would obviously be tricky given we use the same background color without transparency